tools/flavor-convert/main.go

```diff
@@ -0,0 +1,450 @@
+/*
+ *  Copyright (C) 2021 Intel Corporation
+ *  SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/antchfx/jsonquery"
+	"github.com/intel-secl/intel-secl/v3/pkg/lib/host-connector/types"
+	"github.com/intel-secl/intel-secl/v3/pkg/model/hvs"
+)
+
+// EventIDList - define map for event id
+var eventIDList = map[string]string{
```

Comment:
this must have been already defined in flavor library, use it

Reply:
Checked in flavor library, this map is not defined there.

Reply:
They are defined in the trust-agent at https://github.com/mjeyarax/go-trustagent/blob/fd6164ef3b685042fc5f4ab1064f0113c9d1b489/eventlog/common.go#L187

Reply:
In go-trust agent source, eventNameList map is defined which has key - "id" and value - "eventName". 
But in flavor convert tool, eventIdList map is defined which has key - "eventName" and the value - "id". ( We are getting the event name from label field-old flavor part json and mapping it with the id to get type_id value in flavor convert tool.)